### PR TITLE
AB#4831 -- Added review and submit initial page

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
### Description
Created the initial review-and-submit page UI behind MSCA

### Related Azure Boards Work Items
[AB#4831](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/4831)

### Screenshots (if applicable)
image

### Additional Notes
TODO: implement submission to MSCA backend to obtain confirmation code

### CODE REVIEW

File: frontend/public/locales/fr/protected-renew.json [hunk 1]
```diff
@@ -431,5 +431,15 @@
       "provincial-benefit-required": "Sélectionnez si vous avez des prestations dentaires provinciales ou territoriales",
       "provincial-territorial-required": "Sélectionnez une province ou un territoire"
     }
+  },
+  "review-submit": {
+    "page-title": "Vérifiez vos renseignements",
+    "form-instructions": "Vous soumettez une demande de renouvellement pour les personnes suivantes :",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/protected-renew.json:437]:
```suggestion
    "form-instructions": "Vous soumettez une demande de renouvellement pour les personnes suivantes\u00a0:",
```

File: frontend/app/routes/protected/renew/$id/review-and-submit.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  const fetcher = useFetcher<typeof action>();
+  const isSubmitting = fetcher.state !== 'idle';
+
+  const applicantName = `${clientApplication.applicantInformation.firstName} ${clientApplication.applicantInformation.lastName}`;
+
+  return (
+    <div className="max-w-prose">
+      <p className="mb-4">{t('protected-renew:review-submit.form-instructions')}</p>
+      <div className="my-6 space-y-2">
+        <li>{applicantName} </li>
```
Comment (nicholas-ly) [inline frontend/app/routes/protected/renew/$id/review-and-submit.tsx:88]:
```suggestion
        <li>{applicantName}</li>
```

File: frontend/app/routes/protected/renew/$id/review-and-submit.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        {children.map((child) => {
+          const childName = `${child.firstName} ${child.lastName}`;
+          return <li key={childName}>{childName}</li>;
+        })}
+      </div>
+      <h2 className="font-lato text-lg font-bold"> {t('protected-renew:review-submit.submit-renewal-title')}</h2>
```
Comment (nicholas-ly) [inline frontend/app/routes/protected/renew/$id/review-and-submit.tsx:94]:
```suggestion
      <h2 className="font-lato text-lg font-bold">{t('protected-renew:review-submit.submit-renewal-title')}</h2>
```

File: frontend/public/locales/fr/protected-renew.json (hunk 1 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/protected-renew.json]: added unicode

File: frontend/app/routes/protected/renew/$id/review-and-submit.tsx (hunk 2 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/app/routes/protected/renew/$id/review-and-submit.tsx]: fixed spacing

File: frontend/app/routes/protected/renew/$id/review-and-submit.tsx (hunk 3 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/app/routes/protected/renew/$id/review-and-submit.tsx]: fixed spacing

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**